### PR TITLE
UI Frame Updates and Routing Implementation

### PR DESCRIPTION
**Pull request by GarrettMoore22**

### CODE REVIEW

**Review by ErylKenner — APPROVED** [review]

File: tsconfig.json [hunk 1]
```diff
@@ -21,6 +21,7 @@
     "jsx": "react-jsx",
   },
   "include": [
-    "src"
+    "src",
+    "components"
```
Comment (kylebrain) [inline tsconfig.json:25]: Revert

File: src/components/MapScreen.tsx [hunk 2]
```diff
@@ -0,0 +1,11 @@
+import React from "react";
+
+const MapScreen = (): JSX.Element => {
+    return (
+        <div className="MapScreenProps">
```
Comment (kylebrain) [inline src/components/MapScreen.tsx:5]: Change to map-screen, and similar for others

**Review by michaelmartinelli15 — COMMENTED** [review]
Should name className for various screens something other than "Prop"

**Review by kylebrain — CHANGES_REQUESTED** [review]
Few requests

**Review by kylebrain — APPROVED** [review]
LGTM

**Pull request merged**